src/pudl/metadata/resources/eia860.py

```diff
@@ -882,6 +883,18 @@
                 "served_transmission_and_distribution_deferral",
                 "served_voltage_or_reactive_power_support",
                 "stored_excess_wind_and_solar_generation",
+                "is_ac_coupled",
+                "is_dc_coupled",
+                "id_dc_coupled_tightly",
+                "is_independent",
+                "is_transmission_and_distribution_asset_support",
+                "is_direct_support",
```

Comment:
these are all new bool columns. the rest of this table uses `served_` bc that's what's in the eia layout - they all seem to be did "x gen serve y thing in the report year" but these new columns seem more static from the eia descriptions